Makefile

```diff
@@ -193,7 +193,7 @@ bundle: $(KUSTOMIZE) $(OPERATOR_SDK) generate
 			--kustomize-dir=deploy/olm \
 			--package=observability-operator \
 		 	$(BUNDLE_METADATA_OPTS)
-	sed -e 's|<IMG_OBSERVABILITY_OPERATOR>|$(OPERATOR_IMG)|g' \
```

Comment:
This isn't really needed is it?

Reply:
No. Is now fixed. I had trouble running the make bundle in my mac 🍏 as `sed` works differently here.